dbt/models/ccao/schema.yml

```diff
@@ -1,11 +1,32 @@
 sources:
   - name: ccao
+    tags:
+      - load_manual
     tables:
       - name: class_dict
+
```

Comment:
Pre-adding spaces here between resources since they'll soon have additional documentation and keys.